jujugui/static/gui/src/app/components/status/status.js

```diff
@@ -5,23 +5,363 @@
 /** Status React component used to display Juju status. */
 class Status extends React.Component {
 
+  /**
+    Render the component.
```

Comment:
I can't remember the exact reason, but I would argue at this point it would be about consistency.